test/server/data/test-suites/regression-gh-599/testfile.js

```diff
@@ -0,0 +1,28 @@
+import { APIError } from '../../../../../lib/errors/runtime';
+
+fixture `Test`
+    .page `http://example.com`;
+
+var obj = {
+    method1: () => {
+        var err = new APIError('method1');
+
+        // NOTE: to preserve callsite
+        err.isTestCafeError = true;
```

Comment:
Why? Just throw regular error, why do this weird trick assigning `isTestCafeError` to runtime error, why not throw test run error in that case?

Just `throw new Error('yo')`